src/libs/StatusBar/types.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line no-restricted-imports
+import {StatusBar as StatusBarRN} from 'react-native';
+
+type StatusBarExtended = typeof StatusBarRN & {
+    getBackgroundColor(): string | symbol | null;
```

Comment:
```suggestion
    getBackgroundColor(): string | null;
```

No need for `symbol` if you apply the suggestions above.